source/frontend/src/features/projects/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { Button } from 'components/common/buttons/Button';
+import { InlineForm, InlineInput } from 'components/common/form/styles';
+import { Scrollable as ScrollableBase } from 'components/common/Scrollable/Scrollable';
+import styled from 'styled-components';
+
+export const ListPage = styled.div`
```

Comment:
can the styles in this page be reused from the other search/list components?

Reply:
Will address on next PR.